libp2p/NodeTable.cpp

```diff
@@ -69,79 +69,81 @@ void NodeTable::processEvents()
 
 bool NodeTable::addNode(Node const& _node)
 {
-    shared_ptr<NodeEntry> entry;
+    LOG(m_logger) << "Adding node " << _node;
+
+    if (!isValidNode(_node))
+        return false;
```

Comment:
I think it would be good to have a log statement here

Reply:
I think there are already enough logs inside `isValidNode`

Reply:
Ah yes, agreed